lib/hpc/thread1.g

```diff
@@ -168,6 +175,43 @@ BIND_GLOBAL("CopyToRegion", {x,y} -> x);
 BIND_GLOBAL("SetTLDefault", BindThreadLocal);
 BIND_GLOBAL("SetTLConstructor", BindThreadLocalConstructor);
 
+
+BIND_GLOBAL("CHECKED_GET_ATOMIC_LIST", function(l, pos)
+    if IsBound(l[pos]) then
+        return [true, l[pos]];
+    else
+        return [false];
+    fi;
+end);
+
+BIND_GLOBAL("COMPARE_AND_SWAP", function(l, pos, old, new)
+    if not(IsBound(l[pos]) and IS_IDENTICAL_OBJ(l[pos], old) ) then
```

Comment:
add space between `not` and `(`.

Also remove the space before the last `)`; or else, add a space after the first `(`.

Reply:
You could also switch the if and the else bodies, and get rid of the `not` (but leaving it as it is of course is also fine)